packages/gatsby-image/src/index.js

```diff
@@ -74,21 +74,31 @@ const noscriptImg = props => {
   // HTML validation issues caused by empty values like width="" and height=""
   const src = props.src ? `src="${props.src}" ` : `src="" ` // required attribute
   const sizes = props.sizes ? `sizes="${props.sizes}" ` : ``
-  const srcSetWebp = props.srcSetWebp ? `<source type='image/webp' srcSet="${props.srcSetWebp}" ${sizes}/>` : ``
-  const srcSet = props.srcSet ? `<source srcSet="${props.srcSet}" ${sizes}/>` : ``
+  const srcSetWebp = props.srcSetWebp
+    ? `<source type='image/webp' srcSet="${props.srcSetWebp}" ${sizes}/>`
+    : ``
+  const srcSet = props.srcSet
+    ? `<source srcSet="${props.srcSet}" ${sizes}/>`
+    : ``
   const title = props.title ? `title="${props.title}" ` : ``
   const alt = props.alt ? `alt="${props.alt}" ` : `alt="" ` // required attribute
   const width = props.width ? `width="${props.width}" ` : ``
   const height = props.height ? `height="${props.height}" ` : ``
   const opacity = props.opacity ? props.opacity : `1`
   const transitionDelay = props.transitionDelay ? props.transitionDelay : `0.5s`
-  return (`<picture>${srcSetWebp}${srcSet}<img ${width}${height}${src}${alt}${title}style="position:absolute;top:0;left:0;transition:opacity 0.5s;transition-delay:${transitionDelay};opacity:${opacity};width:100%;height:100%;object-fit:cover;object-position:center"/></picture>`)
+
+  // needed for object-fit-images polyfill:
+  var objectFit = props.objectFit ? props.objectFit : `cover`
+  var objectPosition = props.objectPosition ? props.objectPosition : `50% 50%`
```

Comment:
default props should handle fallbacks, also please don't use vars

Reply:
No problem! I was just mirroring the code already in that area. Do you want me to refactor this whole section to use default props and remove vars?